sdk/formrecognizer/Azure.AI.FormRecognizer/tests/Models/FieldValueTests.cs

```diff
@@ -86,7 +86,6 @@ public void InstantiateFieldValueWithTimeAndNoValueTime()
         }
 
         [Test]
-        [Ignore("https://github.com/Azure/azure-sdk-for-net/issues/17814")]
         public void InstantiateFieldValueWithSelectionMarkAndNoValueSelectionMark()
```

Comment:
Found this by chance in Form Recognizer. It seems the issue is closed, and the test passes locally. Let me know if we should keep ignoring it.